generators/chipyard/src/main/scala/clocking/DividerOnlyClockGenerator.scala

```diff
@@ -146,5 +147,5 @@ class DividerOnlyClockGenerator(pllName: String)(implicit p: Parameters, valName
 }
 
 object DividerOnlyClockGenerator {
-  def apply()(implicit p: Parameters, valName: ValName) = LazyModule(new DividerOnlyClockGenerator(valName.name)).node
+  def apply()(implicit p: Parameters, valName: ValName) = LazyModule(new DividerOnlyClockGenerator(valName.name))
```

Comment:
Just delete this helper object if we have to reference the module after construction

Reply:
I'll probably just delete the helper object.